.github/workflows/weekly.yml

```diff
@@ -80,6 +80,24 @@ jobs:
         # See tools/ci/src/main.rs for the commands this runs
         run: cargo run -p ci -- compile
 
+  check-doc:
```

Comment:
The reversion suggested I re-add this, but I'm not sure if that's correct.

Reply:
Yup, that shouldn't have been reverted. If you look at the [original PR](https://github.com/bevyengine/bevy/pull/16147/files#diff-b077dee47bccc37c2b6fef32cdd102f36acadad73ede1f410c3941f39c50cd94), it doesn't remove this snippet from the code.